<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RotateDisp</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/pic/logo.png</normaloff>:/pic/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="4" column="0">
     <widget class="QGroupBox" name="groupBox_RSetting">
      <property name="title">
       <string>Rotation Setting</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_Monitor">
             <property name="text">
              <string>Monitor:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_Monitor">
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_Auto">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="text">
            <string>Automatic setting based on sensor</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_Reverse">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="text">
            <string>Reverse rotation</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QToolButton" name="toolButton_Up">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Ctrl + Up</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/pic/monitor-u.png</normaloff>:/pic/monitor-u.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_Manu</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QToolButton" name="toolButton_Left">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Ctrl + Left</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/pic/monitor-l.png</normaloff>:/pic/monitor-l.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_Manu</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QToolButton" name="toolButton_Right">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Ctrl + Right</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/pic/monitor-r.png</normaloff>:/pic/monitor-r.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_Manu</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QToolButton" name="toolButton_Down">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Ctrl + Down</string>
            </property>
            <property name="text">
             <string>Default</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/pic/monitor-d.png</normaloff>:/pic/monitor-d.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_Manu</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_Device">
      <property name="title">
       <string>GY-25T Settings</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBox_DataBits">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_DataBits">
         <property name="text">
          <string>Data Bits:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_Port">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pushButton_Open">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/pic/logo.png</normaloff>:/pic/logo.png</iconset>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_Baud">
         <property name="text">
          <string>Baud Rate:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_Parity">
         <property name="text">
          <string>Parity:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_Port">
         <property name="text">
          <string>Serial Portï¼š</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBox_Parity">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_StopBits">
         <property name="text">
          <string>Stop Bits:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_Baud">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBox_StopBits">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pushButton_Refresh">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>F5</string>
         </property>
         <property name="text">
          <string>Scan Device</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox_Boot">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="text">
         <string>Auto start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_Hide">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="text">
         <string>Hide</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox_Port</tabstop>
  <tabstop>comboBox_Baud</tabstop>
  <tabstop>comboBox_DataBits</tabstop>
  <tabstop>comboBox_Parity</tabstop>
  <tabstop>comboBox_StopBits</tabstop>
  <tabstop>pushButton_Open</tabstop>
  <tabstop>comboBox_Monitor</tabstop>
  <tabstop>checkBox_Auto</tabstop>
  <tabstop>toolButton_Up</tabstop>
  <tabstop>toolButton_Left</tabstop>
  <tabstop>toolButton_Down</tabstop>
  <tabstop>toolButton_Right</tabstop>
 </tabstops>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_Manu"/>
 </buttongroups>
</ui>
